src/frontend/web_application/src/scenes/DiscussionList/components/DiscussionItemActionsContainer/presenter.jsx

```diff
@@ -66,8 +66,13 @@ class DiscussionItemActionsContainer extends Component {
         >
           {__('discussion-item-actions.action.more')}
         </DropdownController>
-        <Dropdown id={this.dropdownId} position="bottom" closeOnClick>
-          <VerticalMenu>
+        <Dropdown
```

Comment:
> So imo:
>
>  *  className m-dropdown should alway be applied to Dropdown comp. even the raw one;

Agree but don't forget the AppSwitcher component that has very specific styles

> *   .m-dropdown should only contain foundation-dropdown styles (no border, no background, no shadow)

not agree, we should completely drop this mixin and make our own because it adds a lot of things (like border, background) and we should not remove it with override directives ("easy to add, hard to remove"): https://github.com/zurb/foundation-sites/blob/develop/scss/components/_dropdown.scss#L42-L60

>  *  there should be modifiers for stylized Dropdown (ex: --dark for current user menu and application Switcher)

+1

---

Well, we 2 things to deal with in dropdown's stylesheets: 
* the business logic: the positioning (position: absolute, z-index, visibility)
* the styles depending on the context: app switcher, user-menu, discussion-list-more-actions-menu, other-menu

So I propose to refactor the dropdown like this:

* Dropdown only gives business logic
* AppSwitcher extends Dropdown with its special styles
* Add new DropdownMenu component with styles and a standard stylized Button
* the other components will used DropdownMenu and Button

-> If you are ok we can do that in an other PR 
(probably me, because you have enough work with the public site in this sprint)